### DISCUSSION TITLE
node 周刊 收录

### DISCUSSION THREAD

**Discussion by zxhycxq** [coder-hxl/x-crawl#23] [Q&A] [answered]
看到node 周刊收录了这个项目（虽然之前在阮一峰周刊上面也看到过）。想知道是作者自己主动提交的（发邮件吗）还是官方自己发现的？

**Comment by coder-hxl** [comment]
你说的 node 周刊是哪个？阮一峰老师的周刊是自荐过去的。
________________________________
发件人: 你好世界 ***@***.***>
发送时间: 2023年4月19日 17:56
收件人: coder-hxl/x-crawl ***@***.***>
抄送: Subscribed ***@***.***>
主题: [coder-hxl/x-crawl] node 周刊 收录 (Discussion #23)


看到node 周刊收录了这个项目（虽然之前在阮一峰周刊上面也看到过）。想知道是作者自己主动提交的（发邮件吗）还是官方自己发现的？

―

**Reply by zxhycxq** [reply]
https://nodeweekly.com/issues/482  最下面部分

**Reply by coder-hxl** [reply, marked as answer]
感谢提供，这个应该官方自己收录的
